vendor/assets/stylesheets/dvl/core/lists.scss

```diff
@@ -1,4 +1,40 @@
-/* Nested `list-style` types for ordered lists */
+// Default and prose list styles
+
+@mixin formatted_list_ul {
+  li {
+    position: relative;
+    margin: 0 0 $lineHeight ($rhythm * 3);
+    &:before {
+      position: absolute;
+      top: $rhythm;
+      left: $rhythm * -3;
+      right: 100%;
+      content: '';
+      width: $rhythm;
+      height: $rhythm;
+      border: 1px solid $darkestGray;
+      border-radius: 50%;
+    }
+  }
+}
+
+@mixin formatted_list_ol {
+  counter-reset: list;
+  li {
+    position: relative;
+    counter-increment: list;
+    margin: 0 0 $lineHeight $rhythm * 5;
```

Comment:
Yup. The dots in ULs are always a uniform width. The numbers in OLs can
increase to triple digits.

On Thu, Jul 2, 2015 at 10:54 AM, Adam Becker <notifications@github.com>
wrote:

> In vendor/assets/stylesheets/dvl/core/lists.scss
> <https://github.com/dobtco/dvl-core/pull/105#discussion_r33804254>:
>
> > +      right: 100%;
> > +      content: '';
> > +      width: $rhythm;
> > +      height: $rhythm;
> > +      border: 1px solid $darkestGray;
> > +      border-radius: 50%;
> > +    }
> > +  }
> > +}
> > +
> > +@mixin formatted_list_ol {
> > +  counter-reset: list;
> > +  li {
> > +    position: relative;
> > +    counter-increment: list;
> > +    margin: 0 0 $lineHeight $rhythm * 5;
>
> This is intentionally larger than for formatted_list_ul?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dobtco/dvl-core/pull/105/files#r33804254>.
>